<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Entrenamiento</class>
 <widget class="QMainWindow" name="Entrenamiento">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Entrenamiento Supervisado</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>favicon.png</normaloff>favicon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="titulo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>561</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Entrenamiento Supervisado</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>60</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Capturas</string>
    </property>
    <widget class="QPushButton" name="boton_entrenamiento">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Comenzar Obtenci칩n</string>
     </property>
    </widget>
    <widget class="QLabel" name="estado_capturas">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Clasificaci칩n</string>
    </property>
    <widget class="QLabel" name="estado_capturas_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="boton_clasificacion">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Comenzar Casificaci칩n</string>
     </property>
    </widget>
    <widget class="QLabel" name="estado_clasificacion">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>C치mara a emplear</string>
    </property>
    <widget class="QRadioButton" name="radio_hikvision">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Hikvision</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radio_foscam">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Foscam</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
